centos_7_py2py3/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM amd64/centos:7
+
+MAINTAINER Cosmo (hello@cloudify.co)
+
+RUN yum -y install python-devel gcc openssl git libxslt-devel libxml2-devel openldap-devel libffi-devel openssl-devel libvirt-devel python-netaddr
+
+RUN yum install -y python-virtualenv
+
+RUN cd / && \
+    virtualenv py2env && \
+    source py2env/bin/activate && \
+    pip install --upgrade pip==9.0.1 && \
+    pip install --upgrade virtualenv==15.1.0 && \
```

Comment:
No point reinstalling virtualenv here unless there are actually venvs being created by whatever uses this venv.